libraries/AP_Compass/AP_Compass_Calibration.cpp

```diff
@@ -264,28 +269,33 @@ bool Compass::send_mag_cal_progress(const GCS_MAVLINK& link)
 
 bool Compass::send_mag_cal_report(const GCS_MAVLINK& link)
 {
-    uint8_t cal_mask = _get_cal_mask();
+    const mavlink_channel_t chan = link.get_chan();
+
+    for (uint8_t i = 0; i < COMPASS_MAX_INSTANCES; i++) {
+        Priority compass_id = (next_cal_report_idx[chan] + 1) % COMPASS_MAX_INSTANCES;
```

Comment:
Should this be `get_count()` ?

Reply:
14 bytes to add it here, and pull it out, and would be another 14 bytes on the other send. I'm inclined to just pay the runtime cost of hitting the `nullptr` check, but if others disagree I don't mind pulling it out

Reply:
get_count() makes the code bigger, codepaths not crossed often.